functional-tests/build.gradle

```diff
@@ -3,24 +3,25 @@ import org.apache.tools.ant.taskdefs.condition.Os
 plugins {
     id "groovy"
     id "codenarc"
-    id "com.energizedwork.webdriver-binaries" version "1.0"
-    id "net.foragerr.jmeter" version "1.0.10-3.3"
+    id "com.energizedwork.webdriver-binaries" version "1.4"
+    id "net.foragerr.jmeter" version "1.1.0-4.0"
 }
 
 ext {
     // The drivers we want to use
     drivers = ["firefox", "firefoxHeadless", "chrome", "chromeHeadless","ie","edge","safari"]
 
     ext {
-        groovyVersion = '2.4.15'
+        groovyVersion = '2.5.1' // Updating this past 2.5.1 causes spock reports to fail the test execution
         gebVersion = '2.2'
-        seleniumVersion = '3.14.0'
+        seleniumVersion = '3.141.59'
         chromeDriverVersion = '2.44'
```

Comment:
Apparently we had already bumped chromedriver to the latest version (which is 2.44) in a previous PR.